### PR TITLE
#2819-Certify-Correct-Text-Updates

### PR DESCRIPTION
**Pull request by ochiu**
*Issue #:* /bcgov/entity#2819

*Description of changes:*
Text updates to Certify Correct sections

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/views/AnnualReport.vue [hunk 1]
```diff
@@ -157,9 +157,9 @@
             <!-- Certify -->
             <section v-show="entityFilter(EntityTypes.BCOMP) || agmDate || noAgm">
               <header>
-                <h2 id="AR-step-4-header" v-if="entityFilter(EntityTypes.BCOMP)">3. Certify Correct</h2>
-                <h2 id="AR-step-4-header" v-else>4. Certify Correct</h2>
-                <p>Enter the legal name of the current director, officer, or lawyer submitting this Annual Report.</p>
+                <h2 id="AR-step-4-header" v-if="entityFilter(EntityTypes.BCOMP)">3. Certify</h2>
+                <h2 id="AR-step-4-header" v-else>4. Certify</h2>
+                <p>Legal name of person authorized to complete and submit this Annual Report.</p>
```
Comment (severinbeauvais) [inline src/views/AnnualReport.vue]: The changes look fine but I'm concerned about dropping "Enter the..." from this subtext. I'm going to check with Scott and Linda again...

Comment (severinbeauvais) [reply src/views/AnnualReport.vue]: Keep an eye on the ticket... Scott has some ideas and Linda/Kaine need to vet them.

Comment (ochiu) [reply src/views/AnnualReport.vue]: Will do. Thanks!

Comment (ochiu) [reply src/views/AnnualReport.vue]: @severinbeauvais 

updates done please take a look

**Pull request merged**